app/utils/formatters.ts

```diff
@@ -36,3 +36,20 @@ export const formatDate = (value: unknown): string => {
 export const formatFileTimestamp = (date: Date): string => {
   return format(date, "yyyy-MM-dd h.mm.ss a");
 };
+
+/**
+ * Return morning, afternoon, or evening
+ */
+export const formatGreetingTime = (
```

Comment:
This whole greeting thing is maybe a bit too cutesy for the subject matter 🤷

Reply:
i kinda agree, i tested replacing it with "Welcome to the Vision Zero Editor" and then the text below just explains the metrics. but idk maybe they would like the personalization but to me its kinda weird

Reply:
I stated it above, but I agree pretty strongly. I think John's inclination and Rose's suggestion are the ticket.